### PR TITLE
Feature/trip duration

### PR DESCRIPTION
**Pull request by DaveKrauss**

### CODE REVIEW

**Review by ahohnsen — APPROVED** [review]
Good job 🙌🏼

File: components/Trip/index.js [hunk 1]
```diff
@@ -17,18 +18,40 @@ export default function Trip({
...
+      return `starts ${countdownAdjusted}`;
+    }
+    if (countdown.includes("ago") && parsedCountdown < 1) {
+      return "starts today";
+    }
+  }
   return (
     <StyledLink href={`/trips/${id}`}>
       <StyledTrip>
         <StyledImage src={image} width={70} height={70} alt="" />
```
Comment (JessicaLoers) [inline components/Trip/index.js:45]: This is not part of the PR, but I suggest to use a more meaningful alt description for the image to aid with accessibility.

```suggestion
        <StyledImage src={image} width={70} height={70} alt={`Image for trip titled ${title}`} />
```

File: pages/trips/[id]/index.js [hunk 2]
```diff
@@ -136,7 +138,38 @@ export default function DetailsPage() {
...
+    }
+  }
+  const duration = formatDistanceStrict(
+    new Date(trip.startDate),
+    new Date(trip.endDate),
+    {
+      unit: "day",
+      roundingMethod: "floor",
+    }
+  );
```
Comment (JessicaLoers) [inline pages/trips/[id]/index.js:172]: Please see my previous comments regarding `DisplayCountdown()` function the and the points we discussed in the Daily. 💚

**Review by JessicaLoers — CHANGES_REQUESTED** [review]
**Well Done Team 🐸**
Please see my comments and the points we discussed in the Daily this morning.

**Review by JessicaLoers — APPROVED** [review]
**Well done ✨**

**Pull request merged**